### ISSUE TITLE
Ability to add other VHDs

### ISSUE THREAD

**Issue by absolutejam** [VirtualEngine/Lability#218]
Hey guys,

Is it currently possible to add more VHDs to the VMs? I've checked the docs but didn't see anything relevant!

Cheers.

**Comment by iainbrighton** [comment]
Hey @absolutejam - it's here in beta form :wink:. This is only currently available on the dev branch and there's an [example here](https://github.com/VirtualEngine/Lability/blob/dev/Examples/MultipleDiskExample.psd1).

**Comment by absolutejam** [comment]
Awesome! I look forward to this being merged through. Will help with my nested virtualisation labs.

**Comment by brmvfm** [comment]
This is Awesome!
I am a newbie in DevOps , 
We are using Lability from Powershell Gallery testing Nested Hyperconverged with scripts from your last Powershel lconference.
Is that a way to test  Dev Branch in Windows 10 with Hyper-v, instead using Gallery files ?
We wish to test and provide feedback.

**Comment by iainbrighton** [comment]
@brmvfm You can download the dev branch [module zip here](https://codeload.github.com/VirtualEngine/Lability/zip/dev) and manually extract it into your module path (remember to unblock it first 😉).

If you have access to the gallery you can do this:
```powershell
## Module for installing GitHub PowerShell repository modules
Install-Module -Name GitHubRepository -Scope CurrentUser
Import-Module -Name GitHubRepository
Install-GitHubRepository -Owner VirtualEngine -Repository Lability -Branch dev -Clean
```

**Comment by brmvfm** [comment]
Thanks for your help.

**Comment by brmvfm** [comment]
Thanks for your help.

**Comment by iainbrighton** [comment]
@absolutejam I have submitted a [PR](https://github.com/PowerShell/xHyper-V/pull/124) for this functionality into the main xHyper-V DSC resource/module. Once this is merged in, the support will be "official".

**Comment by iainbrighton** [comment]
Merged in #285. Official support is coming in the next release 😃
